lte/gateway/c/oai/tasks/sgw/pgw_handlers.c

```diff
@@ -536,68 +563,67 @@ uint32_t pgw_handle_nw_initiated_bearer_deactv_req(
   OAILOG_FUNC_IN(LOG_PGW_APP);
   MessageDef *message_p = NULL;
   uint32_t i = 0;
-  uint32_t j = 0;
+  uint32_t itrn = 0;
   hash_table_ts_t *hashtblP = NULL;
   uint32_t num_elements = 0;
   s_plus_p_gw_eps_bearer_context_information_t *spgw_ctxt_p = NULL;
   hash_node_t *node = NULL;
   itti_s5_nw_init_deactv_bearer_request_t *itti_s5_deactv_ded_bearer_req = NULL;
-  bool found = false;
+  bool is_lbi_found = false;
+  bool send_reject = false;
+  bool is_imsi_found = false;
+  bool is_ebi_found = false;
+  ebi_t ebi_to_be_established[BEARERS_PER_UE] = {0};
+  uint32_t no_of_bearers_to_be_est = 0;
```

Comment:
more of "ebi_to_be_deactivated" and "no_of_bearers_to_be_deact"

Reply:
done